pulsar-client-cpp/lib/ClientConnection.cc

```diff
@@ -1013,10 +1141,17 @@ void ClientConnection::close() {
     }
 
     // Fail all pending lookup-requests on the connection
-    for (PendingLookupRequestsMap::iterator it = pendingLookupRequests_.begin(); it != pendingLookupRequests_.end(); ++it) {
+    for (PendingLookupRequestsMap::iterator it = pendingLookupRequests_.begin();
+         it != pendingLookupRequests_.end(); ++it) {
         it->second->setFailed(ResultConnectError);
     }
-    
+
+    for (PendingConsumerStatsMap::iterator it = pendingConsumerStatsMap_.begin();
+         it != pendingConsumerStatsMap_.end(); ++it) {
+        LOG_ERROR(cnxString_ << " In getConsumerStats call - Closing Client Connection, please try again later");
```

Comment:
Copy pasted message? Also we shouldn't have to explicitly say "In getConsumerStats call"